utils/uploadToS3.ts

```diff
@@ -0,0 +1,29 @@
+import aws from "aws-sdk";
+import fs from "fs";
+import path from "path";
+
+const uploadToS3 = (filePath: string): void => {
+    aws.config.update({
+        accessKeyId: process.env.AWS_ACCESS_KEY,
+        secretAccessKey: process.env.AWS_SECRET_KEY,
+        region: process.env.AWS_REGION,
+        signatureVersion: "v4",
+    });
+    const s3 = new aws.S3();
```

Comment:
I think `new aws.S3()` should be exported to this file so we don't create new s3 instances

Reply:
I assume you meant exported _from_ this file? If so, see https://github.com/uwblueprint/social-diversity-for-children/pull/11/commits/d683e84f081d2625d8dce69dbf3c462f5be123c8
Otherwise, let me know what you meant

Reply:
After #12 is merged in, perhaps you could set up the s3 object in `aws/index.js`.

Reply:
Actually, I think it might be okay to merge yours in first. Perhaps you can create an `index.js` file where you set up everything from aws. See `aws/index.js` in https://github.com/uwblueprint/social-diversity-for-children/pull/12/files and https://github.com/uwblueprint/dancefest-web/blob/staging/aws/index.js

Reply:
See https://github.com/uwblueprint/social-diversity-for-children/pull/11/commits/4156ac1af5483819aca96900b2314dc72f6a486e